### DISCUSSION TITLE
Membership

### DISCUSSION THREAD

**Discussion by BlossomThemeAdmin** [BlossomTheme/.github#6] [Q&A]
Wanna join the org ?? 
Drop a message including your Github Username & what you would like to do in the organization...
Invitations will be sent out within 24 hours!

**Comment by sachinsenal0x64** [comment]
Hey, I want to create some themes, so I am interested in contributing. :)

**Reply by BlossomThemeAdmin** [reply]
Sure, Invitation Sent...

**Comment by ShamalLakshan** [comment]
Closing this Discussion due to implementation of a better membership requesting system : [Membership](https://github.com/BlossomTheme/BlossomTheme/blob/master/Membership/MEMBERSHIP.md)
(Hope it works fine :) )
